MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -92,13 +98,15 @@ public typealias UnpackCompletionHandler = (_ numberOfTiles: UInt64, _ error: Er
  If the request was canceled or there was an error obtaining the routes, this argument is `nil`. This is not to be confused with the situation in which no results were found, in which case the array is present but empty.
  - parameter error: The error that occurred, or `nil` if the placemarks were obtained successfully.
  */
+@available(*, deprecated, message: "Not recommended to use")
 public typealias OfflineRouteCompletionHandler = (_ session: Directions.Session, _ result: Result<RouteResponse, OfflineRoutingError>) -> Void
 
-/**
+/** :nodoc:
  A `NavigationDirections` object provides you with optimal directions between different locations, or waypoints. The directions object passes your request to a built-in routing engine and returns the requested information to a closure (block) that you provide. A directions object can handle multiple simultaneous requests. A `RouteOptions` object specifies criteria for the results, such as intermediate waypoints, a mode of transportation, or the level of detail to be returned. In addition to `Directions`, `NavigationDirections` provides support for offline routing.
  
  Each result produced by the directions object is stored in a `Route` object. Depending on the `RouteOptions` object you provide, each route may include detailed information suitable for turn-by-turn directions, or it may include only high-level information such as the distance, estimated travel time, and name of each leg of the trip. The waypoints that form the request may be conflated with nearby locations, as appropriate; the resulting waypoints are provided to the closure.
  */
+@available(*, deprecated, message: "Not recommended to use")
```

Comment:
For this particular deprecation, let’s recommend Directions as an alternative. It doesn’t do offline routing, but at least it does routing.

```suggestion
@available(*, deprecated, message: "Use the Directions class instead.")
```